data/StevenBlack/hosts

```diff
@@ -584,6 +584,24 @@
 0.0.0.0 stage.tvpage.com
 0.0.0.0 appcdn.tvpage.com
 0.0.0.0 ad.wynk.in
+0.0.0.0 ads.linkshrink.net
+0.0.0.0 cdn2.contenabc.com
+0.0.0.0 config.mobile.yahoo.com
+0.0.0.0 constintptr.com
+0.0.0.0 greatene.com
+0.0.0.0 gusi11.com
+0.0.0.0 indian-sex-specialist.online
+0.0.0.0 lerold.pro
+0.0.0.0 marketingfresh2018mobile.win
+0.0.0.0 mine-sub-new-techieonline.win
+0.0.0.0 nativeadmatch.com
+0.0.0.0 pop.bid
+0.0.0.0 soundwave.mobile.yahoo.com
+0.0.0.0 steepto.com
+0.0.0.0 uidesign.gbtcdn.com
+0.0.0.0 uod2quk646.com
+0.0.0.0 www.onclicksuper.com
+0.0.0.0 yes-no-click.online
```

Comment:
Just curious, How did you obtain this domain?

Cos when I test it, I get the `SERVFAIL` from 3 different locations

```
dig yes-no-click.online

; <<>> DiG 9.11.3-1ubuntu1.10-Ubuntu <<>> yes-no-click.online
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 819
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;yes-no-click.online.           IN      A

;; Query time: 135 msec
;; SERVER: 127.0.0.53#53(127.0.0.53)
;; WHEN: Wed Nov 13 02:04:14 CET 2019
;; MSG SIZE  rcvd: 48
```

Reply:
It used to be a fake Flash update domain: `https://web.archive.org/web/20180903074530/http://yes-no-click.online/`

Reply:
Hmm so it _used_ to be a malicious site if I understand you @llacb47 currect, for that it could be a vavry good reason to keep it, as the domain is still active :unamused: